front/components/assistant_builder/ActionsScreen.tsx

```diff
@@ -346,6 +347,31 @@ export default function ActionsScreen({
           </div>
         </div>
       </div>
+      <div className="flex flex-col gap-8">
+        {owner.flags.includes("visualization_action_flag") && (
+          <div className="flex flex-row gap-2">
+            <Checkbox
+              checked={builderState.visualizationEnabled}
+              onChange={() => {
+                setEdited(true);
+                setBuilderState((state) => ({
+                  ...state,
+                  visualizationEnabled: !state.visualizationEnabled,
+                }));
+              }}
+              variant="checkable"
+            />
+            <div>
+              <div className="flex w-full gap-1 font-medium text-element-900">
```

Comment:
Can we center the text vertically with the checkbox?

Reply:
On the figma I think it's like this no ? https://www.figma.com/design/QOxHwppG64GtPocpDhS6Y7/Product?node-id=6977-6066&t=pQumT6kuqeKJ6fuE-0

Reply:
Okok